include/mrdox/Support/Dom.hpp

```diff
@@ -96,6 +96,19 @@ class MRDOX_DECL
     */
     Array(impl_type impl) noexcept;
 
+    /** Assignment.
+
+        The newly constructed array will contain
+        references to its structured data.
+    */
+    Array&
+    operator=(Array const& other)
```

Comment:
UGH... is there any way to avoid this? There are consequences for scripts...

Reply:
Assigning to arrays is important because there are many situations where we need to (re)populate arrays to create the callbacks, call helpers, etc... and

```
implicitly-declared ‘clang::mrdox::dom::Array& clang::mrdox::dom::Array::operator=(const clang::mrdox::dom::Array&)’ is deprecated because ‘clang::mrdox::dom::Array’ has user-provided ‘clang::mrdox::dom::Array::Array(const clang::mrdox::dom::Array&)’
   89 |     Array(Array const& other);
```